config.cc

```diff
@@ -2339,10 +2386,11 @@ QString getCacheDir() throw()
 {
   return isPortableVersion() ? portableHomeDirPath() + "/cache"
 #if QT_VERSION >= QT_VERSION_CHECK( 5, 0, 0 )
-                             : QStandardPaths::writableLocation( QStandardPaths::CacheLocation );
+                             : QStandardPaths::writableLocation( QStandardPaths::GenericCacheLocation )
 #else
-                             : QDesktopServices::storageLocation( QDesktopServices::CacheLocation );
+                             : QDesktopServices::storageLocation( QDesktopServices::GenericCacheLocation )
 #endif
+                                + QDir::separator() + CACHE_DIR_NAME;
```

Comment:
There is no such thing as `QDesktopServices::GenericCacheLocation` => move the added code into the Qt5-specific section. Qt4 and Qt5 network caches are incompatible and stored in separate subdirectories, so the name difference shouldn't matter.

And don't use `QDir::separator()`. See https://doc.qt.io/qt-5/qdir.html#separator.

Reply:
Sorry, I should have checked if Qt4 has it.

About `QDir::separator()` – it is used in few other places in this file, but it's good to know you don't have to use it in this case.